src/simd/config.rs

```diff
@@ -17,18 +17,21 @@ pub trait SIMDInstructionSet {
 // ----------------------------- x86_64 / x86 -----------------------------
 
 pub struct SSE;
+pub struct SSEIgnoreNaN;
 
 impl SIMDInstructionSet for SSE {
     const REGISTER_SIZE: usize = 128;
```

Comment:
It would be good to include some source reference on where these register sizes come from. Is there a nice table we can link to as a source of truth for these values?

Reply:
Surprisingly hard to find a good (readable) table online - so I'll plug the wikipedia pages and create a minimal table in the docs!